iohk-monitoring/test/Cardano/BM/Test/Mock.lhs

```diff
@@ -48,18 +49,18 @@ instance IsEffectuator MockSwitchboard a where
 \end{code}
 
 \subsubsection{traceMock}\label{code:traceMock}\index{traceMock}
-A |Tracer| which forwards |LogObject|s to |MockSwitchboard| simulating
+A |Trace| which forwards |LogObject|s to |MockSwitchboard| simulating
 functionality of |mainTraceConditionally|.
 
 \begin{code}
-traceMock :: MockSwitchboard a -> Config.Configuration -> Tracer IO (LogObject a)
+traceMock :: MockSwitchboard a -> Config.Configuration -> Trace IO a
 traceMock ms config =
-    Tracer $ \item@(LogObject loname _ _) -> do
-        traceWith mainTrace item
-        subTrace <- fromMaybe Neutral <$> Config.findSubTrace config (loname2text loname)
+    Tracer $ \(ctx, lo) -> do
```

Comment:
It's not really `ctx`, more `loName`.

Reply:
we are looking up the `Subtrace` type in the configuration with the context name, and this is provided in the `Trace`